### ISSUE TITLE
Navbar covering the search input field on desktop screens

### ISSUE THREAD

**Issue by zyzer01** [neokd/DataStorehouse#118]
## Description
The search input field is being covered by the navbar on desktop, which hinders the users from seeing what they are searching
 
## Expected Behavior
The search input field should be scooped down a bit to ensure proper viewing.

## Current Behavior
Currently, the navbar covers the search input field

## Screenshots or Additional Information
![Screenshot from 2023-10-04 12-14-56](https://github.com/neokd/DataStorehouse/assets/82843086/40efba24-89c6-45de-b938-fbe0bff9e1c5)

## Environment
Desktop

I am willing to work on this issue.

**Comment by neokd** [comment]
@Zyzer01 we are developing a new website with nextjs the current deployed one is deprecated you can check the website [here ](https://github.com/neokd/DataStorehouse/tree/main/Website)

**Comment by Bchass** [comment]
> @Zyzer01 we are developing a new website with nextjs the current deployed one is deprecated you can check the website [here ](https://github.com/neokd/DataStorehouse/tree/main/Website). You can still work on this issue by developing a new search bar for the website. Also are you working under hacktoberfest or its just a open source contribution?

Is there a timeline for when the new website will be deployed to production?

**Comment by neokd** [comment]
Yes, we are expecting it to launch by end of October

**Comment by Zyzer01** [comment]
> @Zyzer01 we are developing a new website with nextjs the current deployed one is deprecated you can check the website [here ](https://github.com/neokd/DataStorehouse/tree/main/Website). You can still work on this issue by developing a new search bar for the website. Also are you working under hacktoberfest or its just a open source contribution?

Yes, I am working under hacktoberfest, I need those labels!

Also, is it possible I work with you on the new website even before launch?

**Comment by neokd** [comment]
Yes you can work on it. The overall goal is to develop the entire new website during hacktoberfest
